modules/docker/init.zsh

```diff
@@ -0,0 +1,35 @@
+#
+# Defines docker and boot2docker commands & aliases
+#
+# Authors:
+#   andrew williams (https://github.com/skarfacegc)
+#
+
+# Only setup the boot2docker commands and aliases if we're on OSX
+if [[ "$OSTYPE" == darwin* ]]; then
```

Comment:
There are native Mac and Windows clients in beta.  They still use a VM behind the scenes, but once these clients leave beta, boot2docker will basically be out of a job.